frontend/app/components/redesign/components/ToolsWalletAddress.tsx

```diff
@@ -168,17 +172,17 @@ export const ToolsWalletAddress = () => {
           <div className="flex-1 min-w-0 h-12">
             <InputField
               ref={inputRef}
-              id="wallet-address-url"
+              id={generatedId}
```

Comment:
Consider using a constant ID for this input, as it'll help with autocomplete. The random generatedId will not give us autocomplete like we want.